scipy/interpolate/_bsplines.py

```diff
@@ -1165,7 +1165,9 @@ def make_interp_spline(x, y, k=3, t=None, bc_type=None, axis=0,
     --------
 
     Use cubic interpolation on Chebyshev nodes:
+
```

Comment:
Because of no this line, the doc rendering is broken:
Before
<img width="707" alt="SS 2022-08-17 at 21 43 15" src="https://user-images.githubusercontent.com/3813847/185135772-415b8a1d-910d-4c66-bfa2-f1052bd96dcf.png">
After:
<img width="894" alt="SS 2022-08-17 at 21 43 42" src="https://user-images.githubusercontent.com/3813847/185135933-db34d3e2-40a1-41c6-af49-884bd47b7a17.png">